package.json

```diff
@@ -178,7 +178,7 @@
     },
     {
       "path": "commerce-sdk-isomorphic-with-deps.tgz",
-      "maxSize": "300 kB"
+      "maxSize": "410 kB"
```

Comment:
had to adjust the package artifact size limit due to the types added.

However, I noticed every namespaced types somehow includes `ShopperLogin` class definition. I'm not sure why, I don't see any import statement that is related to `ShopperLogin`...

Reply:
update, @wjhsf helped figure out a bad import that caused the bloat.